src/index.js

```diff
@@ -289,7 +298,7 @@ class Dropzone extends React.Component {
         onDragLeave={this.onDragLeave}
         onDrop={this.onDrop}
       >
-        {this.props.children}
+        {this.renderChildren()}
```

Comment:
Ok. I can do both, tell which you prefer ;) In my opinion, `static` makes more sense (that way it is scoped, etc), but it's an advanced feature of Javascript, so it makes it less readable for beginners. I'll do whichever you think is best.